deeplearning4j/deeplearning4j-scaleout/spark/dl4j-spark/src/main/java/org/deeplearning4j/spark/iterator/SparkADSI.java

```diff
@@ -108,7 +107,6 @@ public SparkADSI(DataSetIterator iterator, int queueSize, BlockingQueue<DataSet>
 
     @Override
     protected void externalCall() {
-        TaskContextHelper.setTaskContext(context);
```

Comment:
This looks a bit suspicious. Why should externalCall be a noop here?

Reply:
It's out of date from spark. Somehow this was left over.